### PR TITLE
Switch 'update available' action from 3rd party code to gh tool

### PR DESCRIPTION
**Pull request by taurus-forever**
DRAFT for testing only.

### CODE REVIEW

**Comment by taurus-forever** [comment]
@shayancanonical what is the best way to test this without merging?

File: .github/workflows/on_bundle_update_available.yaml [hunk 1]
```diff
@@ -11,26 +11,20 @@ jobs:
...
-          draft: false
+        run: gh pr create \
+          --title 'Update Bundles' \
+          --body 'Update bundles with new revisions from Charmhub.' \
+          --base main
+          --assignee team-reviewers, shayancanonical, paulomach \
+          --reviewer team-reviewers, shayancanonical, paulomach \
+          --label 'automated pr'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (shayancanonical) [inline .github/workflows/on_bundle_update_available.yaml:30]: minor nit: missing new line

**Review by shayancanonical — COMMENTED** [review]
@taurus-forever would it be sufficient to test by attaching an upterm session (once there is a divergence in bundle component revisions) and running the bash commands manually on the ssh session?

**Comment by taurus-forever** [comment]
Abandoned as no longer topical. We are using shared workflow now.

**Pull request closed without merging**